test/success.test.js

```diff
@@ -2083,8 +2198,72 @@ test('Skip commention on issues/PR if "successComment" is "false"', async (t) =>
   );
 
   t.true(
-    t.context.log.calledWith("Skip commenting on issues and pull requests."),
+    t.context.log.calledWith("Added labels %O to issue #%d", ["released"], 1),
+  );
+
+  t.true(fetch.done());
+});
+
+test('Skip comment on issues/PR and skip label if "successComment" is "false" / "releaseLabels" is "false"', async (t) => {
```

Comment:
```suggestion
test('Skip comment on issues/PR and skip label if "successComment" is "false" / "releasedLabels" is "false"', async (t) => {
```

Can you please fix the other test names as well?

Reply:
other than that this is good to go, thank you @kristof-mattei!